bmaptools/BmapHelpers.py

```diff
@@ -37,6 +45,7 @@ def human_size(size):
 
     return "%.1f %s" % (size, 'EiB')
 
+
```

Comment:
Why do you add these newlines? I think the unspoken convention in this code is to have double newlines between classes and only single between functions. It may be silly, and not big deal, but it is what it is. It is better to be consistent, so the whole code base would look coherent and thus, easier to follow.

If you like 2 newlines, you can suggest a _separate_ pull request cleaning this up globally, not just in one file. All or nothing.